src/app/reporting/reporting-default-configuration.c

```diff
@@ -62,6 +65,28 @@ void emberAfPluginReportingLoadReportingConfigDefaults(void)
 #endif
 }
 
+/** @brief Configured
+ *
+ * This callback is called by the Reporting plugin to get the default reporting
+ * configuration values from user if there is no default value available within
+ * af generated default reporting configuration tabel. The application need to
+ * write to the minInterval, maxInterval and reportable change in the passed
+ * IO pointer in the arguement while handleing this callback, then application
+ * shall return true if it has provided the default values or else false for
+ * reporting plugin to further handleing.
+ *
+ * @param entry   Ver.: always
+ */
+bool emberAfPluginReportingGetDefaultReportingConfigCallback(EmberAfPluginReportingEntry * entry)
```

Comment:
Ported from Silabs SDK

Reply:
This info should probably be in the commit message.